build/es5.umd/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Do we need all these tsconfig files? If we can have one file (as small as possible) in the root of the project, that would be ideal.